salt/modules/virt.py

```diff
@@ -1959,6 +2003,61 @@ def init(
                       1: 10
                       2: 21
                       3: 31
+               tuning:
+                    vcpupin:
+                      0: 1-4,^2  # vcpuid : cpuset
+                      1: 0,1
+                      2: 2,3
+                      3: 0,4
+                    emulatorpin: 1-3
+                    iothreadpin:
+                      1: 5,6    # iothread id: cpuset
+                      2: 7,8
+                    shares: 2048
+                    period: 1000000
+                    quota: -1
+                    global_period: 1000000
+                    global_quota: -1
+                    emulator_period: 1000000
+                    emulator_quota: -1
+                    iothread_period: 1000000
+                    iothread_quota: -1
+                    vcpusched:
+                      0:        # vcpus id
+                        scheduler: fifo
+                        priority: 1
+                      1:
+                        scheduler: fifo
+                        priority: 2
+                      2:
+                        scheduler: rr
+                        priority: 3
+                    iothreadsched:
+                        iothreads: 2
+                        scheduler: batch
```

Comment:
This doesn't match the implementation. Should be like this:

```diff
-                        iothreads: 2
-                        scheduler: batch
+                       0:
+                          iothreads: 2
+                          scheduler: batch
```

But even this way, how do we handle the default then where the `iothread id` is not defined? Same question for `vcpusched`